New version: NMRScreen v0.0.4

JuliaRegistrator opened the pull request:
- Registering package: NMRScreen
- Repository: https://github.com/waudbygroup/NMRScreen.jl
- Created by: @chriswaudby
- Version: v0.0.4
- Commit: e03d21f869caeb969f4ef05bae68f5c91216c236
- Reviewed by: @chriswaudby
- Reference: https://github.com/waudbygroup/NMRScreen.jl/commit/e03d21f869caeb969f4ef05bae68f5c91216c236#commitcomment-152426699
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
- there are no breaking changes
- ability to navigate back to registration
- documentation added
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd667dbdfad787ba91c663604e3e3581abca8ec9601e89f5da05b46209dd04bb3c3e4c219e8605a5a1b586116b2dc5d608e271ab383cbd4bdc281d1e2efa7cca87edf2156484e8594d34b4b755f5a4f5aec09ba8b2ebd959e7da4e84e4fc2a247541629b467792f40d239ad1a56b0345380988f449b1f26717ed2b80d1e56fa6febfa3ecedb6d0e41ee5066001b14b90d945a55fd413448ea8bfb5d47cb9c429f27b30e3fa9a9378fd6584cf19c51b085ae -->

chriswaudby commented:
@JuliaRegistrator register

Release notes:

- There are no breaking changes
- You can now navigate back to the first registration window
- Press shift + [],. to snap to maxima in registration window

JuliaRegistrator committed:
New version: NMRScreen v0.0.4

UUID: e6af4f88-7701-4df4-8af7-4e9c6957a0a0
Repo: https://github.com/waudbygroup/NMRScreen.jl.git
Tree: 477d01ba64f11c7bf5c4208d5f20f5b05fc61f72

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
